data/strings/types_strings.txt

```diff
@@ -4427,6 +4442,7 @@
     it = Punto di ritiro
     mr = पार्सल लॉकर
     nl = Pakketjeskluis
+    pl = Automat paczkowy
```

Comment:
```suggestion
    pl = Paczkomat```

Reply:
Wiesz dobrze, o co chodzi. Chcę uniknąć zamieszania 😁
Jaka jest Twoja argumentacja tej zmiany?

Reply:
To że paczkomat to standardowy termin i "automat paczkowy" używany jest tylko tam gdzie inne firmy zostały zmuszone przez błędny wyrok lub tam gdzie pracownicy inPostu wymusili stosowanie dziwnego terminu w interesie swojego pracodawcy.

Nie widzę sensu by temu ulegać, wręcz przeciwnie.